lib/atscppapi/src/Transaction.cc

```diff
@@ -98,6 +97,46 @@ Transaction::~Transaction()
   delete state_;
 }
 
+bool Transaction::configIntSet(TSOverridableConfigKey conf, int value) 
+{
+    //txn,conf,value
+    TSReturnCode res=TSHttpTxnConfigIntSet(state_->txn_,conf,(TSMgmtInt)value);
+    return (res==TS_SUCCESS);
+}
+bool Transaction::configIntGet(TSOverridableConfigKey conf, int *value)
+{
+    TSReturnCode res=TSHttpTxnConfigIntGet(state_->txn_, conf, (TSMgmtInt*)value);
+    return (res==TS_SUCCESS);
+}
+
+bool Transaction::configFloatSet(TSOverridableConfigKey conf, float value) 
+{
+    //txn,conf,value
+    TSReturnCode res=TSHttpTxnConfigFloatSet(state_->txn_,conf,(TSMgmtFloat)value);
+    return (res==TS_SUCCESS);
+}
+bool Transaction::configFloatGet(TSOverridableConfigKey conf, float *value)
+{
+    TSReturnCode res=TSHttpTxnConfigFloatGet(state_->txn_, conf, value);
+    return (res==TS_SUCCESS);
+}
+
+bool Transaction::configStringSet(TSOverridableConfigKey conf, char* value,int length) 
```

Comment:
The string methods should have `std::string const&` as the argument type. Using common C++ types is a big part of the reason for having this API.